### DISCUSSION TITLE
Thanks man!

### DISCUSSION THREAD

**Discussion by Alquemist** [hegdepavankumar/Cisco-Images-for-GNS3-and-EVE-NG#16] [General]
;-)

**Comment by loicloyal** [comment]
SVP, COMMENT RÉSOUDRE CE PROBLÈME DE FIXATION DES PERMISSIONS sur EVE-NG???

/opt/unetlab/wrappers/unl_wrapper -a fixpermissions
PHP Remarque : file_get_contents() : La lecture de 8192 octets a échoué avec errno=21 Est un répertoire dans /opt/unetlab/html/includes/init.php à la ligne 71
